docs/documentation/pages/DECKHOUSE_CONFIGURE.md

```diff
@@ -3,65 +3,89 @@ title: "How to configure?"
-- `<moduleName>Enabled` (where `<moduleName>` is the name of the Deckhouse module in camelCase) — this one explicitly [enables or disables the module](#enabling-and-disabling-the-module).
+- `metadata.name` — the name of the resource is the name of the Deckhouse module (kebab-cased).
+- `spec.version` — a version of module settings.
+- `spec.settings` — an object with module settings.
+- `spec.enabled` — optional boolean value to explicitly [enable or disable the module](#enabling-and-disabling-the-module). The module may be enabled by default depending on the [used bundle](#module-bundles) when the parameter is not set.
 
-Use the following command to view the `deckhouse` ConfigMap:
+If `spec.settings` is not empty, `spec.version` is required. The latest version number is available in the description of module settings.
 
-```shell
-kubectl -n d8-system get cm/deckhouse -o yaml
-```
+Settings version may become obsolete with new releases. The Deckhouse will support previous versions to allow managing ModuleConfig resources using IaC. Also, it will warn about necessity to update `spec.settings` and `spec.version` when resource is changed or viewed.
+
+Resource `ModuleConfig/global` stores global settings. "global" can't be disabled, so the value of `spec.enabled` is ignored.
 
-Example of the `deckhouse` ConfigMap:
+Example of `ModuleConfig` resources:
 
 ```yaml
-apiVersion: v1
+apiVersion: deckhouse.io/v1
+kind: ModuleConfig
 metadata:
-  name: deckhouse
-  namespace: d8-system
-data:
-  global: |          # Note the vertical bar.
-    # Section of the YAML file with global settings.
+  name: global
+spec:
+  version: 1
+  settings:
     modules:
       publicDomainTemplate: "%s.kube.company.my"
-  # monitoring-ping related section of the YAML file.
-  monitoringPing: |
+---
+# monitoring-ping settings.
+apiVersion: deckhouse.io/v1
+kind: ModuleConfig
+metadata:
+  name: monitoring-ping
+spec:
+  version: 1
+  settings:
     externalTargets:
     - host: 8.8.8.8
-  # Disabling the dashboard module.
-  dashboardEnabled: "false"
+---
+# Disable the dashboard module.
+apiVersion: deckhouse.io/v1
+kind: ModuleConfig
+metadata:
+  name: dashboard
+spec:
+  enabled: false
 ```
 
-Pay attention to the following:
-- The `|` sign — vertical bar glyph that must be specified when passing settings, because the parameter being passed is a multi-line string, not an object.
-- A module name is in *camelCase* style.
+`status` field contains module state, you can get module state after applying changes:
+
+```shell
+kubectl get moduleconfigs
+NAME                VERSION   AGE   ENABLED              STATUS
+deckhouse           1         12h   Enabled              Ready
+deckhouse-web       2         12h   Enabled              Ready
+global              1         12h   Always On
+prometheus          2         12h   Enabled              Ready
+upmeter             2         12h   Disabled by config
+```
```

Comment:
We have changed printer columns. This example is not actual

Reply:
Done